lib/src/home/home_page.dart

```diff
@@ -1,25 +1,86 @@
 import 'package:flutter/material.dart';
```

Comment:
Move this file to the `app` directory because the home directory will contain nothing else.